src/main/java/com/rog/EShop/services/ItemExportService.java

```diff
@@ -0,0 +1,75 @@
+package com.rog.EShop.services;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.sql.*;
+
+@Service
+public class ItemExportService {
+    @Value("${spring.datasource.url}")
+    String url;
+    @Value("${spring.datasource.username}")
+    String username;
+    @Value("${spring.datasource.password}")
+    String password;
+
+    public void exportCSV() {
+
+        try (Connection connection = DriverManager.getConnection(url, username, password)) {
+            String sql = "SELECT * FROM items";
```

Comment:
You mixed here Spring Boot and standard JDBC. It's not ok. Instead of JDBC use Use spring data jpa `ItemRepository.findAll()`